modules/mod_ginger_rdf/models/m_rdf.erl

```diff
@@ -20,6 +20,8 @@
 
 m_find_value(#rdf_resource{} = Rdf, #m{value = undefined} = M, _Context) ->
     M#m{value = Rdf};
+m_find_value(Id, #m{value = undefined}, Context) when is_integer(Id) ->
+    mochijson2:encode(ginger_json_ld:serialize(to_triples(Id, Context)));
```

Comment:
@ddeboer I think this will be a problem for when you are using a model to chain a query. For example m.rdf[id].predicate will not work anymore? You could use the to_value function to realise the actual value and return and #m record here instead.